cove_ocds/tests.py

```diff
@@ -1003,16 +999,7 @@ def test_cove_ocds_cli_schema_cache():
     assert sorted(results['extensions']['invalid_extensions'], key=lambda k: k[0])[1][1] == '404: not found'
 
     # Use cached schema from previous call_command
-    cache_time_total = 0
-
-    for i in range(6):
```

Comment:
I'd quite like to keep this test if we can.

I note that `range(6)` has 6 items `[0, 1, 2, 3, 4, 5]`, but the average below is calculated by dividing by 5. Perhaps aligning these will be sufficient to stop the test failing?

Reply:
@Bjwebb well spotted! That may be it. I'll fix it.